src/content/reference/platform/language.md

```diff
@@ -0,0 +1,63 @@
+
+# Languages
+
+## Background
+
+Workers is a polyglot platform. You can likely write code on Workers with a language you already know. We originally built Workers to operate on JavaScript and WASM and over time we've added support for new languages. We are excited to support and deepen integrations with popular languages like Python for Workers.
+
+## JavaScript / TypeScript
+
+JavaScript is fully supported on the Workers platform. We recommend using JS. You can also use TypeScript on the platform and can find the [type definitions here](https://github.com/cloudflare/workers-types).
+
+<TableWrap>
+
+| Language     | Example Project | 
+|--------------|--------|
+| JavaScript   |   https://github.com/cloudflare/worker-template |
+| TypeScript   |   https://github.com/EverlastingBugstopper/worker-typescript-template     |
+
+</TableWrap>
+
+### See Also:
+- [`TypeScript Type Definitions`](https://github.com/cloudflare/workers-types)
+- [`Hello World for Workers in JS`](https://github.com/cloudflare/worker-template)
+- [`Javascript and Webstandard APIs`](/reference/runtime-apis/web-standards)
+- [`Starters - Code snippets`](/starters)
+- [`Tutorials`](/tutorials)
+
+## WASM-supported
+
+WebAssembly -- often abbreviated as "WASM" -- is a technology that extends the web platform to support compiled languages like C, C++, Rust, Go, and more. Since these languages can be compiled to a special WASM binary format and then loaded in a browser, you can use them on the Workers platform!  Learn more by checking out the example projects.
```

Comment:
I think mentioning Go is misleading given that we have no examples in our docs.

Reply:
yeah that makes sense, will remove it